_source/_posts/2020-04-20-android-authentication.adoc

```diff
@@ -0,0 +1,508 @@
+- "This post walks you through creating a new Android app with OAuth 2.0 auth💥"
+image: blog/featured/okta-android-skew.jpg
+type: conversion
+---
+:toc: macro
+:page-liquid:
+:experimental:
+
+With mobile apps becoming ever-present in users' lives, following best security practices has become essential in protecting your users and your apps. Implementing security alone and from the ground up can be costly for your development team, create a less-than-ideal user experience, and be susceptible to design/implementation errors. The easiest path to strong security, and a positive user experience, is to turn to the experts.
+
+The Okta OIDC SDK can help with this in many ways:
+- Our security experts take the development effort of creating a secure authentication method off your plate
+- OAuth 2.0 and OpenID Connect allows your users to easily authenticate on your app with social login, using industry-standard secure protocols
+
+This post will show you a simple example of how you can use Okta OIDC SDK to create a (Kotlin) Android app with authentication.
+
+toc::[]
+
+== (TLDR) Download the finished app with Android Authentication
+
+If you want to follow along using a completed version of the app,clone this repository from  https://github.com/oktadeveloper/okta-oidc-android-example[GitHub] using the `simplified` branch:
+
+[source,sh]
+----
+git clone https://github.com/oktadeveloper/okta-oidc-android-example.git -b simplified
+cd okta-oidc-android-example
+----
+
+NOTE: The finished app in the `master` branch from the repository includes everything described in this blog, along with  some additional UI and functionality, testing Okta OIDC easier, and providing you with a production-level skeleton to work with when creating your own app.
+
+== Creating your first Android app
+
+First, you'll need to download and install https://developer.android.com/studio[Android Studio].
+Next, launch the app and navigate  to **File** → **New...** → **New Project...**. Then, create an "Empty Activity" for "Phone and Tablet". You should now see a screen similar to this:
+
+image::{% asset_path 'blog/android-authentication/android-studio-new-project.png' %}[alt=Create new Android Project,width=700,align=center]
+
+Choose a "Package name" related to a domain you own (I own akaita.com, therefore I used `com.akaita.myapplication` in the example), a **Save location** you can remember, "Kotlin", and a "Minimum API level" of `23`. Finally, click **Finish**.
+
+[#okta-oidc-config]
+== Get your Okta OIDC configuration settings
+
+Before adding anything else to your app, you'll need to quickly obtain the configuration parameters required to configure Okta OIDC authentication:
+
+1. Log in to your Okta Developer account or https://developer.okta.com/[sign up for a **free** developer account]
+2. Once you've logged on, you'll see the Admin Dashboard. Take a note of your "Org URL" (at the top right of the screen). In my case, I see: `\https://dev-123456.okta.com`
```

Comment:
```suggestion
2. Once you've logged on, you'll see the Admin Dashboard. Take note of your "Org URL" (at the top right of the screen). In my case, I see: `https://dev-123456.okta.com`
```

Reply:
The backslack prevents the automatic href creation

Reply:
(i'll remove the extra `a` though)